examples/fabric-admin/rpc/RpcServer.cpp

```diff
@@ -86,7 +97,10 @@ class FabricAdmin final : public rpc::FabricAdmin, public IcdManager::Delegate
     pw::Status OpenCommissioningWindow(const chip_rpc_DeviceCommissioningWindowInfo & request,
                                        chip_rpc_OperationStatus & response) override
     {
-        NodeId nodeId                    = request.node_id;
+        VerifyOrReturnValue(request.has_id, pw::Status::InvalidArgument());
+        // TODO: OpenDeviceCommissioningWindow uses the same controller every time and doesn't currently accept FabricIndex. For now
```

Comment:
Create github issue for properly tracking this and add the issue number to this TODO comment